### PR TITLE
Release geopandas 0.2

### PR DESCRIPTION
**Pull request by jorisvandenbossche**

### CODE REVIEW

**Comment by jorisvandenbossche** [comment]
@ocefpaf It is correct that I have to set the build number to 0 again?

**Comment by ocefpaf** [comment]
> @ocefpaf It is correct that I have to set the build number to 0 again?

Yep. That is the best practice. Thanks for the PR!

**Comment by jorisvandenbossche** [comment]
seems the pypi download url has changed

**Comment by ocefpaf** [comment]
I am on my phone no but I'll send to you the new URLs we are using. That long and ugly one is terrible to maintain.

**Comment by jorisvandenbossche** [comment]
@ocefpaf Two questions:

- is that the correct url? (it seems such a long url should not be necessary if I look at other packages)
- py3.5 build is failing. Is this expected? It gives again a similar error as before "he following specifications were found to be in conflict"

**Comment by jorisvandenbossche** [comment]
Ah, so the question on the url you already started to answer :-)

**Comment by ocefpaf** [comment]
> Windows py3.5 build is failing. Is this expected? It gives again a similar error as before "The following specifications were found to be in conflict"

Not sure what is going on. I have to investigate all the dependencies to see what is going awry. However, if you are OK without PY35 we can merge this as-is and fix it later.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
 
 package:
     name: geopandas
     version: {{ version }}
 
 source:
     fn: geopandas-{{ version }}.tar.gz
-    url: https://pypi.python.org/packages/source/g/geopandas/geopandas-{{ version }}.tar.gz
-    md5: a01b5d3eb6bc097665a0ba297c39029f
+    url: https://pypi.python.org/packages/2a/1e/ea0c9ed982a62d693cbebda0da10321361a88d2a44c67292f64ca3675275/geopandas-{{ version }}.tar.gz
```
Comment (ocefpaf) [inline recipe/meta.yaml]: Let's use the more elegant `https://pypi.io/packages/source/g/geopandas/geopandas-{{ version }}.tar.gz` :wink:

**Comment by ocefpaf** [comment]
Ping @emiliom who is looking forward for this release.

File: recipe/meta.yaml (hunk 1 above)
Comment (jorisvandenbossche) [inline recipe/meta.yaml]: thanks, that's cleaner!

**Comment by jorisvandenbossche** [comment]
OK, everything is passing now (except for the py35 on windows), so merging

**Comment by jorisvandenbossche** [comment]
OK, all packages are up! I will open an issue so we don't forget the py35 windows problem.

**Comment by ocefpaf** [comment]
Thanks @jorisvandenbossche!

**Comment by emiliom** [comment]
Woohoo! Thanks for the ping

**Pull request merged**